al-khaser/AntiDebug/LowFragmentationHeap.cpp

```diff
@@ -51,11 +51,16 @@ Return Value:
 #endif
 	}
 
+	// When running in a Win10 VM FrontEndHeap does not get initialized,
+	// so we get a null pointer dereferencing error.
+	if (FrontEndHeap == NULL) {
+	        return TRUE;
```

Comment:
Good catch! Shouldn't we return FALSE in this case ? Because this means we will return that the debugger was found any time somebody run the tool on Win10.

So maybe we can do something like:
if  (FrontEndHeap && *FrontEndHeap == NULL) return TRUE;

Reply:
My bad! Fixed the fix.